glitter/utils.py

```diff
@@ -101,3 +102,16 @@ def duplicate(obj, value=None, field=None, duplicate_order=None):
             if root_obj is None:
                 root_obj = obj
     return root_obj
+
+
+def django_version_gt_18():
```

Comment:
Drop this function, just use:

```
if django.VERSION >= (1, 8)
```

In any code which needs to figure out the running version of Django.